__tests__/app.test.js

```
@@ -66,6 +66,40 @@ describe("GET /api/reviews", () => {
   });
 });
 
+describe("GET /api/reviews/:review_id", () => {
+  it("should respond with a single review object", () => {
+    const expectedReview = {
+      review_id: 1,
+      title: "Agricola",
+      designer: "Uwe Rosenberg",
+      owner: "mallionaire",
+      review_img_url:
+        "https://images.pexels.com/photos/974314/pexels-photo-974314.jpeg?w=700&h=700",
+      review_body: "Farmyard fun!",
+      category: "euro game",
+      created_at: "2021-01-18T10:00:20.514Z",
+      votes: 1,
+    };
+
+    return request(app)
+      .get("/api/reviews/1")
+      .expect(200)
+      .then(({ body }) => {
+        const { review } = body;
+        expect(review).toMatchObject(expectedReview);
+      });
+  });
+  it("should respond with a 204 status code if no review is found", () => {
```

Statuses in the 200s imply the request was successful, "review not found" should return a different error code.